docs/website/01_framework/assembler.md

```diff
@@ -83,12 +88,17 @@ The available FE requirements are explained on the [FE requirements](feRequireme
 It offers the functions of [VectorFlatAssembler](#vectorflatassembler) plus more, like:
 
 ```cpp
-Eigen::SparseMatrix<double> &getRawMatrix(const FERequirementType &feRequirements)
-Eigen::SparseMatrix<double> &getMatrix(const FERequirementType &feRequirements)
-Eigen::SparseMatrix<double> &getReducedMatrix(const FERequirementType &feRequirements)
+Eigen::SparseMatrix<double> &matrix(const FERequirementType &feRequirements,MatrixAffordance affo, EnforcingDBCOption qt)
```

Comment:
ampersand belongs to type (i think the formatting is a bit off in the markdown files, we should stay consistent)
```suggestion
Eigen::SparseMatrix<double>& matrix(const FERequirementType& feRequirements, MatrixAffordance affo, EnforcingDBCOption qt)
```